community/server/src/test/java/org/neo4j/server/modules/ThirdPartyJAXRSModuleTest.java

```diff
@@ -54,18 +54,20 @@ public void shouldReportThirdPartyPackagesAtSpecifiedMount() throws Exception
         when( neoServer.getDatabase() ).thenReturn( mock(Database.class));
         
         ConfigurationBuilder configurator = mock( ConfigurationBuilder.class );
-        when( neoServer.getConfigurationBuilder() ).thenReturn( configurator );
+//        when( neoServer.getConfigurationBuilder() ).thenReturn( configurator );
 
         Config config = mock( Config.class );
         List<ThirdPartyJaxRsPackage> jaxRsPackages = new ArrayList<ThirdPartyJaxRsPackage>();
         String path = "/third/party/package";
         jaxRsPackages.add( new ThirdPartyJaxRsPackage( "org.example.neo4j", path ) );
         when( config.get( ServerSettings.third_party_packages ) ).thenReturn( jaxRsPackages );
-        when( neoServer.getConfigurationBuilder().configuration() ).thenReturn( config );
+//        when( neoServer.getConfigurationBuilder().configuration() ).thenReturn( config );
         
         ThirdPartyJAXRSModule module = new ThirdPartyJAXRSModule(webServer, config, NullLogProvider.getInstance(), neoServer );
         module.start();
 
         verify( webServer ).addJAXRSPackages( any( List.class ), anyString(), anyCollection() );
+
+        throw new RuntimeException( "Ha" );
```

Comment:
"Ha", apparently. I left that there to remind myself to fix that test before I made this PR, evidently that did not work.. :)